modules/olm-installplan.adoc

```diff
@@ -5,4 +5,98 @@
 [id="olm-installplan_{context}"]
 = Install plan
 
-An _install plan_, defined by an `InstallPlan` object, describes a set of resources to be created to install or upgrade to a specific version of an Operator, as defined by a cluster service version (CSV).
+An _install plan_, defined by an `InstallPlan` object, describes a set of resources to be created by Operator Lifecycle Manager (OLM) for installing or upgrading to a specific version of an Operator, as defined by a cluster service version (CSV).
+
+To install an Operator, a cluster administrator, or a user who has been granted Operator installation permissions, must first create an `Subscription` object, which represents the intent to subscribe to a stream of available versions of an Operator from a catalog source. The subscription then creates an `InstallPlan` object to facilitate the installation of the resources for the Operator.
+
+The install plan must then be approved according to one of the following approval strategies:
+
+* If the subscription's `sub.spec.installPlanApproval` field is set to `Automatic`, the install plan is approved automatically.
```

Comment:
Suggest that remove the `sub`, only `spec.installPlanApproval` is enough I guess.